### ISSUE TITLE
Uncomplete tasks are deleted

### ISSUE THREAD

**Issue by wmahfoudh** [lumoe/obsidian-rollover-daily-todos#119]
Hello,
I noticed today that when creating a new daily note, unchecked tasks from previous daily note are added and then deleted. Not sure what might be the cause. 
Using the Daily Notes plugin, and the Blue Topaz theme if this could help. All plugins last versions.
Thanks.

**Comment by czottmann** [comment]
Same here. 

```
SYSTEM INFO:
	Obsidian version: v1.4.14
	Installer version: v1.4.13
	Operating system: Darwin Kernel Version 22.6.0: Fri Sep 15 13:41:28 PDT 2023; root:xnu-8796.141.3.700.8~1/RELEASE_ARM64_T6000 22.6.0
	Login status: logged in
	Catalyst license: supporter
	Insider build toggle: on
	Live preview: off
	Legacy editor: off
	Restricted mode: off
	Plugins installed: 32
	Plugins enabled: 30
		1: Actions URI v1.3.0
		2: Advanced Tables v0.19.1
		3: Better footnote v1.0.1
		4: Blockquote Levels v1.1.0
		5: Contextual Typography v2.2.5
		6: Copy Search URL v1.2.1
		7: Dataview v0.5.59
		8: Dynamic Table of Contents v0.0.27
		9: File Explorer Note Count v1.2.1
		10: Hotkey Helper v0.3.18
		11: Hotkeys for specific files v1.4.0
		12: Native Scrollbars v1.0.1
		13: Obsidian Git v2.22.0
		14: Settings Search v1.3.9
		15: Share as Gist v1.3.1
		16: Simple Embeds v1.15.0
		17: Style Settings v1.0.7
		18: Table to CSV Exporter v0.1.4
		19: Tag Wrangler v0.6.1
		20: Templater v1.16.3
		21: Periodic Notes v0.0.17
		22: Rollover Daily Todos v1.1.7
		23: Recent Files v1.3.8
		24: Tasks v4.9.0
		25: Linter v1.20.0
		26: Outliner v4.8.0
		27: Omnisearch v1.17.1
		28: Auto Link Title v1.4.1
		29: Task Marker v0.4.3
		30: Mononote v1.0.0
```

**Comment by czottmann** [comment]
Update: For me, this _seems_ to be fixed with in Obsidian v1.4.15. Yesterday, the update was applied, and this mornings the plugin worked as expected.

**Comment by wmahfoudh** [comment]
Issue resolved by updating Obsidianto version 1.4.16
